api-doc/api-doc.yaml

```diff
@@ -3976,7 +3920,7 @@ paths:
   /v1/programs/addRolesInScope/{programId}:
```

Comment:
@MallanagoudaB Please verify if we are using this api right now. I think we are using enity add remove apis for role related scope changes as well